### ISSUE TITLE
Parcels variable nonres_building_sqft ignores mixed-use residential sqft

### ISSUE THREAD

**Issue by stefancoe** [psrc/urbansim2#171]
I assume the intent of this variable is not to get the sum of only the non-residential buildings. It's current form is this:

```python
@orca.column('parcels', 'nonres_building_sqft', cache=True, cache_scope='iteration')
def nonres_building_sqft(parcels, buildings):
    return (buildings.building_sqft * (~buildings.is_residential)).groupby(buildings.parcel_id).sum().\
           reindex(parcels.index).fillna(0)
```

I have created a nonres_building_sqft on the buildings table so I'll re-write this to sum by bulldings.non_residential_sqft by parcel_id

**Comment by hanase** [comment]
@stefancoe This variable seems to be used only in the non-res REPM (configs/repmnr.yaml). I suppose its purpose is to look at the total size of non-res buildings, regardless, if there are any  res units or not. Here again, its a problem of naming. Since this variable is only used directly in specs, it should be a matter of a significance, i.e. is this variable stronger or weaker than if we'd use non-res sqft only. It will be probably very similar.

**Comment by stefancoe** [comment]
@hanase Right- If possible, it would be ideal to have 'unit' variables only include residential space. This would avoid a lot of confusion.

**Comment by stefancoe** [comment]
@hanase For now, I am adding a comment to the variable to clarify that it is only for non-res buildings. It's no longer being used in the hlcm.
